File: 02.conditions/cpp/08.cpp
#include <iostream>

using namespace std;

int main(){
    int c_m, g_m, c_s, g_s, rate;
    cin >> c_m >> g_m >> c_s >> g_s >> rate;

    if (c_m + g_m * rate >= c_s + g_s * rate){
        cout << "Yes" << endl;
    }else{
        cout << "No" << endl;
    }

    return 0;
}

File: 02.conditions/python/08.py
c_m, g_m = map(int, input().split()) 
c_s, g_s = map(int, input().split())
rate = int(input())

if c_m + g_m * rate >= c_s + g_s * rate:
    print("Yes")
else:
    print("No")
